### PR TITLE
OPSEXP-2909 Remove checkout step from verify-compose and enable specifying path to postman tests

### PR DESCRIPTION
**Pull request by pmacius**
### Checklist

- Jira Reference (also in PR title): OPSEXP-2909
- [README](https://github.com/Alfresco/alfresco-build-tools/blob/master/docs/README.md) updated after adding/changing behaviour of an action
- Proposed version increment for [release](https://github.com/Alfresco/alfresco-build-tools/blob/master/docs/README.md#release):
  - [ ] Patch (bugfix)
  - [x] Minor (new feature)
  - [ ] Major (breaking changes)
- External PR link where changes has been tested: https://github.com/Alfresco/alfresco-dockerfiles-bakery/pull/89

### Description

Remove checkout on action to be able to use compose files produced before running step with this action
Enable specifying path to postman tests

### CODE REVIEW

File: .github/actions/dbp-charts/verify-compose/action.yml [hunk 1]
```diff
@@ -25,10 +25,6 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Checkout
-      uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
-      with:
-        fetch-depth: 0
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-compose/action.yml:31]: if there was fetch-depth then it should had something to do with parsing commits, but I don't see anymore which could rely on that so ok to drop it

File: .github/actions/dbp-charts/verify-compose/docker_compose.sh [hunk 2]
```diff
@@ -3,7 +3,6 @@
 COMPOSE_FILE=$(basename $COMPOSE_FILE_PATH)
 COMPOSE_PATH=$(dirname $COMPOSE_FILE_PATH)
 COMPOSE_BIN="docker compose"
-alf_port=8080
 
 dump_all_compose_logs() {
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-compose/docker_compose.sh:7]: we have docker-dump-containers-logs action, we can drop this here and then decide if to integrate it here or rely on upstream workflow to call it

**Review by gionn — CHANGES_REQUESTED** [review]

File: version.txt [hunk 3]
```diff
@@ -1 +1 @@
-v8.0.0
+v8.0.1
```
Comment (pmacius) [inline version.txt:1]:
```suggestion
v8.1.0
```

File: .github/actions/dbp-charts/verify-compose/docker_compose.sh [hunk 4]
```diff
@@ -1,36 +1,20 @@
 #!/bin/bash -e
 
-COMPOSE_FILE=$(basename $COMPOSE_FILE_PATH)
-COMPOSE_PATH=$(dirname $COMPOSE_FILE_PATH)
 COMPOSE_BIN="docker compose"
-alf_port=8080
+POSTMAN_PATH=$(realpath "${POSTMAN_PATH}")
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-compose/docker_compose.sh:4]: variable is used only once so you can also inline it in docker run

File: .github/actions/dbp-charts/verify-compose/docker_compose.sh [hunk 5]
```diff
@@ -1,36 +1,20 @@
...
 fi
 export COMPOSE_HTTP_TIMEOUT=120
-$COMPOSE_BIN -f "${COMPOSE_FILE}" up -d --quiet-pull --wait
+$COMPOSE_BIN -f "${COMPOSE_FILE_PATH}" up -d --quiet-pull --wait
 
 echo "All services are up and running... starting postman tests"
 
-cd ..
-docker run -a STDOUT --volume "${PWD}"/test/postman/docker-compose:/etc/newman --network host postman/newman:5.3 run "acs-test-docker-compose-collection.json" --global-var "protocol=http" --global-var "url=localhost:8080"
+docker run -a STDOUT --volume "${POSTMAN_PATH}":/etc/newman --network host postman/newman:5.3 run "${POSTMAN_JSON}" --global-var "protocol=http" --global-var "url=localhost:8080"
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-compose/docker_compose.sh:16]:
```suggestion
docker run -a STDOUT --volume "${POSTMAN_PATH}:/etc/newman" --network host postman/newman:5.3 run "${POSTMAN_JSON}" --global-var "protocol=http" --global-var "url=localhost:8080"
```

more readable if they wrap the whole argument - even if not really strictly needed

File: .github/actions/dbp-charts/verify-compose/docker_compose.sh [hunk 6]
```diff
@@ -1,36 +1,20 @@
...
 
 retVal=$?
 
-[ "${retVal}" -eq 0 ] && echo "Postman tests were successful" || dump_all_compose_logs
+[ "${retVal}" -eq 0 ] && echo "Postman tests were successful" || echo "Postman tests failed"
```
Comment (gionn) [inline .github/actions/dbp-charts/verify-compose/docker_compose.sh:20]: get rid of these as well - not really providing any help

File: version.txt (hunk 3 above)
Comment (gionn) [inline version.txt:1]:
```suggestion
v8.1.0
```

**Comment by pmacius** [comment]
tested after review: https://github.com/Alfresco/alfresco-dockerfiles-bakery/actions/runs/11501325679/job/32013861389

**Pull request merged**